Userland/Libraries/LibWeb/CSS/Parser/DeclarationOrAtRule.h

```diff
@@ -37,7 +38,7 @@ class DeclarationOrAtRule {
         return *m_declaration;
     }
 
-    DeprecatedString to_deprecated_string() const;
+    ErrorOr<String> to_string() const;
```

Comment:
:thinking: is this unused? Why keep it?

Reply:
:thinking: It does seem to be. I think `Rule::to_string()` is only used by this too, so that would go away as well. My inclination is to keep "convenient for dumping things when debugging" stuff like this but I don't know if that's actually reasonable.

Reply:
I've removed it. Less code = better code!